packages/handbrake.rb

```diff
@@ -9,6 +9,14 @@ class Handbrake < Package
   when 'x86_64'
     source_url 'https://github.com/HandBrake/HandBrake/releases/download/1.3.3/HandBrake-1.3.3-source.tar.bz2'
     source_sha256 '218a37d95f48b5e7cf285363d3ab16c314d97627a7a710cab3758902ae877f85'
+
+    binary_url ({
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/handbrake-1.3.3-1-chromeos-x86_64.tar.xz',
+  })
+    binary_sha256 ({
+      x86_64: '13e06c6458fe918ab1a46deeadcea9415c00be877a333b9bfbe5969c1ca1ba2a',
+  })
```

Comment:
Can we fix the alignment of `})`?  It helps clarify and prevent confusion.

Reply:
That would be ideal... I haven't modified my scripts to generate the URL section properly when all architectures haven't been built.

Feel free to edit it?  I won't be able to get back to this until tomorrow...